### PR TITLE
Guidelines and templates

### PR DESCRIPTION
**Pull request by cpswan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+   ``` sh
+   # show how to install the tools needed to work with the code here
+   ```
+
+
+### GitHub Repository Clone
+
+To prepare your dedicated GitHub repository:
+
+1. Fork in GitHub https://github.com/atsign-foundation/at_client_sd
```
Comment (murali-shris) [inline CONTRIBUTING.md]: 'k' missing at the end

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+2. Clone *your forked repository* (e.g., `git clone git@github.com:yourname/at_client_sd`)
```
Comment (murali-shris) [inline CONTRIBUTING.md]: 'k' missing at the end

Comment (cpswan) [reply CONTRIBUTING.md]: well spotted - should be fixed in latest commit

Comment (cpswan) [reply CONTRIBUTING.md]: Typo in URLs was fixed with https://github.com/atsign-foundation/at_client_sdk/pull/147/commits/5a4e16348cfc08f5a154d50782b2943010cc007a so are we good to go with this now?

**Pull request merged**